narratives/ncov_sit-rep_ja_2020-03-13.md

```diff
@@ -85,7 +85,7 @@ COVID-19とそれを引き起こすウイルスであるSARS-CoV-2を理解す
 
 ## Nextstrain からの状況報告
 
-このあとのページが [Nextstrain](https://nextstrain.org) を使用して行われた解析です。左側のサイドバーをスクロールすると項目ごとのテキストが表示され、右側に対応するゲノム情報が表示されます。
+このあとのページが [Nextstrain](https://nextstrain.org) を使用して行われた解析です。スクロールすると項目ごとのテキストと対応するゲノム情報が表示されます。
```

Comment:
It seems that core members made these pages responsive
and remove the words like `left` and `right` because, when you narrow window size, the page is rendered as single column layout

(by the way, this is awesome work)

Reply:
I agree. The previous version did not support the mobile environment very well.